Acornfile

```diff
@@ -0,0 +1,51 @@
+name:        "Rust Sample Acorn"
+description: "Acorn running a sample Rust app"
+readme:      "./README.md"
+
+
+args: {
+	// Name of the database to create. Defaults to "instance"
+	dbName: "rust-sqlx"
+}
+
+services: db: {
+    image: "ghcr.io/acorn-io/postgres:v15.#-#"
+    serviceArgs: {
+        dbName: args.dbName
+    }
+}
+
+containers: {
+  backend: {
+    image: "ghcr.io/infracloudio/rust-backend:v1"
+    env: {
+    "POSTGRES_HOST": "@{service.db.address}"
+    "POSTGRES_PORT": "@{service.db.port.5432}"
+    "POSTGRES_USER": "@{service.db.secrets.admin.username}"
+    "POSTGRES_PASSWORD":"@{service.db.secrets.admin.password}"
+    "POSTGRES_DB":"@{service.db.data.dbName}"
+    "DATABASE_URL":"postgresql://@{service.db.secrets.admin.username}:@{service.db.secrets.admin.password}@@{service.db.address}:5432?schema=public"
+    "PGADMIN_DEFAULT_EMAIL":"admin@admin.com"
+    "PGADMIN_DEFAULT_PASSWORD":"password123"
+    }
+    ports: {
+      publish: "8000:8000/http"
+    }
+    memory: 1048Mi
+    cmd: ["/bin/sh", "-c", "cd /usr/src/fullstack-rust-app/backend && sqlx migrate run; cargo run --bin backend -- --port 8000"]
+    consumes: ["db"]
+  }
+  frontend: {
+    image: "ghcr.io/infracloudio/rust-frontend:v1"
+    cmd: ["/bin/sh", "-c", "cd /usr/src/fullstack-rust-app/frontend && BACKEND_SERVER=$BACKEND_SERVER trunk serve --address 0.0.0.0 --port 3000"]
```

Comment:
@aashimodi14 Same question as above ☝🏼.

Reply:
yes, we have dependency on the backserver URL and it needs to be passed during runtime as its a wasm application and for yew to properly update the variable.